### PR TITLE
Update links and references to Grafana

### PR DESCRIPTION
**Pull request by gguillotte-grafana**
Per grafana/grafana-json-datasource#311 and grafana/grafana-csv-datasource#222 by @yesoreyeram, update links and references to Grafana Labs' maintenance. CC @wbrowne 

(plugin.json `id` is left unchanged to match the above PRs.)

### CODE REVIEW

**Review by wbrowne — COMMENTED** [review]
LGTM other than one little thing - it looks like we can do the same as we did with the `json` data source:

https://github.com/grafana/grafana-json-datasource/pull/311#issuecomment-1278947676

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 2.0.1 (2022-10-14)
+
+[Full changelog](https://github.com/grafana/grafana-treemap-panel/compare/v2.0.0...v2.0.1)
+
+- Fix broken docs and links.
```
Comment (darrenjaneczek) [inline CHANGELOG.md:7]: Perhaps we can leave a remark here noting the change to `grafana-treemap-panel`

**Comment by tolzhabayev** [comment]
I guess this has to be picked up by someone else @grafana/docs-squad

**Review by darrenjaneczek — APPROVED** [review]

**Comment by GrafanaWriter** [comment]
Thanks, @tolzhabayev - we'll get this one taken care of.

**Review by Eve832 — APPROVED** [review]

**Pull request merged**